hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryEntryFactory.java

```diff
@@ -24,13 +24,14 @@
 
 public final class QueryEntryFactory {
     private final boolean useCache;
+    private final SerializationService serializationService;
```

Comment:
Could you please align the field and constructor parameter order if you touch this PR anyway?

![Pleaaaze](http://uploads.neatorama.com/images/posts/109/67/67109/1384912856-1.jpg)